ani-cli

```diff
@@ -307,11 +308,14 @@ episode_selection () {
 		if [ "$is_download" -eq 1 ]; then
 			inf "Range of episodes can be specified: start_number end_number"
 		fi
-		prompt "Choose episode" "[$first_ep_number-$last_ep_number]"
-		ep_choice_start=$REPLY
-		ep_choice_end=$REPLY2
-		is_half_ep=$(cut -c1-1 "$ep_choice_start")
-		if [ "$is_half_ep" = "h" ]
+		if [ -z "$ep_choice_to_start" ]; then
+			prompt "Choose episode" "[$first_ep_number-$last_ep_number]"
+			ep_choice_start=$REPLY
+			ep_choice_end=$REPLY2
+		else
+			ep_choice_start=$ep_choice_to_start && unset ep_choice_to_start
+		fi
+		if [ "$(echo "$ep_choice_start" | cut -c1-1)" = "h" ]
```

Comment:
- [ ] Evaluate the cut command outside the if condition (SC2312)
- [ ] Unnescessary echo into cut. Just give the argument directly to cut

Reply:
Did it